### DISCUSSION TITLE
DiscordUtils

### DISCUSSION THREAD

**Discussion by ghost** [nextcord/nextcord#263] [Q&A] [answered]
i need the library but i use nextcord, is there NextcordUtils library?

**Comment by ooliver1** [comment, marked as answer]
No there is not, DiscordUtils should work with nextcord

**Reply by ghost** [reply]
but nextcord blocks discord.py library btw i use repl it

**Reply by TAG-Epic** [reply]
Install it, then uninstall discord.py then reinstall nextcord. Solved!

**Reply by ghost** [reply]
oh let me try

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/94598301/142374751-34a691fd-c243-4f97-9279-a54e8715f6d3.png)
shows me this but yk what im done

**Reply by VincentRPS** [reply]
did you reinstall nextcord?

**Reply by ghost** [reply]
oh- i forgot😂

**Reply by ghost** [reply]
forgot to comment back btw its works thanks to you and the others
